OpenOversight/app/email_client.py

```diff
@@ -96,7 +27,7 @@ def __new__(cls, dev=False, testing=False):
                 SERVICE_ACCOUNT_FILE, scopes=cls.SCOPES
             )
             delegated_credentials = credentials.with_subject(
-                BaseConfig.OO_SERVICE_EMAIL
+                current_app.config["OO_SERVICE_EMAIL"]
```

Comment:
This should address this build error: https://github.com/lucyparsons/OpenOversight/actions/runs/5488540812